packages/liveblocks-core/src/room.ts

```diff
@@ -486,8 +489,8 @@ type CommentsApi<M extends BaseMetadata> = {
     | undefined
   >;
   createThread(options: {
-    threadId: string;
-    commentId: string;
+    threadId?: string;
```

Comment:
We're setting `threadId` and `commentId` from `@liveblocks/react` to be able to reconcile optimistic updates. I made them optional to simplify the default usage. How do you feel about this @nvie?

Reply:
Are all of these valid/meaningful calls?

1. `createThread()`
2. `createThread({ threadId })`
3. `createThread({ threadId, commentId })`
4. `createThread({ commentId })`

I don't think (2) and (4) should ever be used, right? I don't think we need to solve/protect against that at the type level, though. But I would like it if we'd add a runtime check there. Either both are undefined, or both are specified.

Reply:
I can change that! Thank you!

Reply:
Agreed with @nvie here, it doesn't make sense to pass just one, good catch!